icechunk-python/python/icechunk/distributed.py

```diff
@@ -34,7 +41,10 @@ def merge_sessions(
     *sessions: Session | list[Session] | list[list[Session]],
     axis: Any = None,
     keepdims: Any = None,
+    computing_meta: bool = False,
 ) -> Session:
+    if computing_meta:
+        return np.array([object()], dtype=object)
```

Comment:
turns out the reduction API is a bit odd in the sense that there is an intermediate blockwise layer and a final tree reduction layer. This meta is used in the tree reduction but not the intermediate. Therefore, the meta computation still triggers once. This can be bypassed with a magical kwarg in the respective functions.